### PR TITLE
Fix: Remove the ignore errors of `View` in phpstan.

### PR DESCRIPTION
**Pull request by ya861220**
**Description**
See #7731 

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/View/Cell.php [hunk 1]
```diff
@@ -233,6 +233,8 @@ final protected function renderCell(BaseCell $instance, string $method, array $p
      * Returns the values from $params that match the parameters
      * for a method, in the order they are defined. This allows
      * them to be passed directly into the method.
+     *
+     * @return array
```
Comment (MGatner) [inline system/View/Cell.php:237]: This method is private so it is safe to apply the native return type.

File: system/View/Exceptions/ViewException.php [hunk 2]
```diff
@@ -15,36 +15,57 @@
 
 class ViewException extends FrameworkException
 {
+    /**
+     * @return ViewException
```
Comment (MGatner) [inline system/View/Exceptions/ViewException.php:19]: Here and below please use the polymorphic form:
```suggestion
     * @return static
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Thank you for working on these errors! A few changes noted, if you please.

File: system/View/Cell.php (hunk 1 above)
Comment (ya861220) [reply system/View/Cell.php]: Ok.

File: system/View/Exceptions/ViewException.php (hunk 2 above)
Comment (ya861220) [reply system/View/Exceptions/ViewException.php]: Ok.

**Review by paulbalandan — APPROVED** [review]

**Review by kenjis — APPROVED** [review]
LGTM!

**Pull request merged**